tests/screenshot/apps/components/Dropdown.js

```diff
@@ -13,20 +13,36 @@ const DropdownTests = [
 	<Dropdown title="Select your option" width="huge">{['Option 1', 'Option 2', 'Option 3']}</Dropdown>,
 
 	// Dropdown Open
-	<Dropdown open title="Select your option">{['Option 1', 'Option 2', 'Option 3']}</Dropdown>,
+	<div>
+		<Dropdown open title="Select your option" style={{marginLeft: '12px'}}>{['Option 1', 'Option 2', 'Option 3']}</Dropdown>
```

Comment:
![image](https://user-images.githubusercontent.com/34676178/112957202-2bb22700-917c-11eb-8827-73cf605a4937.png)
I don't know why the dropdown looks like this,
It seems that `develop` branch has the same problem.
 let's fix it in another issue instead of marginLeft. Please make jira issue and remove thease marginLeft

Reply:
Yes, maybe a separate issue is better. The popup position issue is related to ContextualPopup rather than Dropdown. I will create a ticker